its/src/test/java/com/sonar/it/csharp/Tests.java

```diff
@@ -83,6 +87,20 @@ public static Path projectDir(TemporaryFolder temp, String projectName) throws I
     return tmpProjectDir;
   }
 
+  static BuildResult analyzeProject(TemporaryFolder temp, String projectName, @Nullable String profileKey, String... keyValues) throws IOException {
```

Comment:
_Optionally:_ Can you add second method with almost the same content with extra subProjectName argument and `TestUtils.createBeginStep(projectName, projectDir, subProjectName)`? That should simplify almost all of the rest of the invocations.